app/model/team.py

```diff
@@ -31,6 +32,7 @@ def get_attachment(self):
             ('Github Team Name', self.github_team_name),
             ('Display Name', self.display_name),
             ('Platform', self.platform),
+            ('Folder', self.folder),
```

Comment:
You will also need to modify the `Team.to_dict()` function so that we can properly store the folder.

Reply:
c5d65981dd8967ead7789563feea61b5c2159f28